platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -92,6 +93,9 @@ object Capture {
         // This is a lazy property to avoid the need to initialize the main thread handler unless needed here
         private val mainThreadHandler by lazy { MainThreadHandler() }
 
+        @Volatile
+        private var crashReported = false
```

Comment:
Having this tracked /w a separate volatile means that we're risking some race conditions but the only other option I see is to synchronize it all which introduces a mutex to the logging path